database/migrations/2020_08_12_192945_add_id_to_experience_skills_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddIdToExperienceSkillsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('experience_skills', function (Blueprint $table) {
+            $table->bigIncrements('id');
```

Comment:
Is there a reason we're adding this now? I can definitely see it making our lives easier but it would've saved me a bunch of time on the review page 😆